pywbemtools/pywbemcli/_pywbem_server.py

```diff
@@ -322,27 +323,38 @@ def get_password(self, ctx):
 
     def to_dict(self):
         """
-        Create dictionary from instance for persisting the connection
-        """
-        dict_ = {"name": self.name,
-                 "server_url": self.server_url,
-                 "user": self.user,
-                 "password": self.password,
-                 "default_namespace": self.default_namespace,
-                 "timeout": self.timeout,
-                 "no_verify": self.no_verify,
-                 "certfile": self.certfile,
-                 "keyfile": self.keyfile,
-                 "ca_certs": self.ca_certs,
-                 "use_pull": self.use_pull,
-                 "pull_max_cnt": self.pull_max_cnt,
-                 "mock_server": self.mock_server,
-                 "log": self.log}
-        return dict_
+        Create dictionary from instance for persisting the connection. All
+        key names that include _ are modified to - so that the dictionary
+        reflects the general option names.
+        """
+
+        return OrderedDict({"name": self.name,
+                            "server": self.server,
+                            "user": self.user,
+                            "password": self.password,
+                            "default-namespace": self.default_namespace,
+                            "timeout": self.timeout,
+                            "no_verify": self.no_verify,
+                            "certfile": self.certfile,
+                            "keyfile": self.keyfile,
+                            "ca-certs": self.ca_certs,
+                            "use-pull": self.use_pull,
+                            "pull-max_cnt": self.pull_max_cnt,
+                            "mock-server": self.mock_server,
+                            "log": self.log})
```

Comment:
See earlier comment about hyphens vs underscores in YAML property names.

Reply:
See above comment.  ** DISCUSSION**

Reply:
I can live with using hyphens, but there were a few consistency issues where underscores were still used. I'll open seoarate comments on them.